core/src/main/java/org/web3j/tx/gas/DefaultGasProvider.java

```diff
@@ -14,12 +14,9 @@
 
 import java.math.BigInteger;
 
-import org.web3j.tx.Contract;
-import org.web3j.tx.ManagedTransaction;
-
 public class DefaultGasProvider extends StaticGasProvider {
-    public static final BigInteger GAS_LIMIT = Contract.GAS_LIMIT;
-    public static final BigInteger GAS_PRICE = ManagedTransaction.GAS_PRICE;
+    public static final BigInteger GAS_LIMIT = BigInteger.valueOf(4_300_000);
```

Comment:
Should we make the default closer to 9_000_000 or 10_000_000, as that seems to be closer to the block limit these days..?

Reply:
I think that is a good idea.